src/layman/common/prime_db_schema/publications.py

```diff
@@ -231,6 +231,12 @@ def get_publication_infos_with_metainfo(workspace_name=None, pub_type=None, styl
                                    '_file_type': file_type,
                                    '_style_type': style_type,
                                    'image_mosaic': image_mosaic,
+                                   '_table_uri': ConnectionString(
+                                       url=f'postgresql://{settings.LAYMAN_PG_USER}:{settings.LAYMAN_PG_PASSWORD}@{settings.LAYMAN_PG_HOST}:{settings.LAYMAN_PG_PORT}/{settings.LAYMAN_PG_DBNAME}',
+                                       schema=workspace_name,
+                                       table=f'layer_{uuid}',
```

Comment:
It's not raw UUID, see get_table_name in layer.db or https://github.com/LayerManager/layman/pull/748/files#diff-dd775c3c3bcba2016f947d1dc08aa9492a0f6ef61f5c30141ba283048ea0307aR218